settings.gradle.kts

```diff
@@ -24,6 +24,13 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-
 rootProject.name = "ProtoData"
 
+include("compiler")
+
+dependencyResolutionManagement {
+    repositories {
+        mavenLocal()
+        jcenter()
```

Comment:
Please add Maven Central too.

Reply:
JCenter will be shut down in less than 3 months (see the official announcement [here](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)). I guess it worth not mentioning it at all.